webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
```

Comment:
There is currently a few bugs with the `@woocommerce/dependency-extraction-webpack-plugin` (note `@woocommerce` instead of `@wordpress`) which meant I couldn't use it here (which would be preferable). However I have a [fix up in the WooCommerce Admin repo for this](https://github.com/woocommerce/woocommerce-admin/pull/6295) that I've tested and works here so once that is published ideally you'll switch to it here and then you can get rid of a lot more config and switch to this for the webpack config:

```js
const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
const DependencyExtractionWebpackPlugin = require( '@woocommerce/dependency-extraction-webpack-plugin' );
module.exports = {
	...defaultConfig,
	plugins: [
		...defaultConfig.plugins.filter(
			( plugin ) =>
				plugin.constructor.name !== 'DependencyExtractionWebpackPlugin'
		),
		new DependencyExtractionWebpackPlugin(),
	],
};
```

Reply:
I pushed e063f34 which implements the newest version of the webpack plugin allowing us to do this 🎉